engine/runtime-compiler/src/main/java/org/enso/compiler/pass/analyse/TailCall.java

```diff
@@ -206,6 +207,16 @@ public Expression transformExpression(Expression ir) {
           // Note [Call Argument Tail Position]
           p.arguments().foreach(a -> markAsTail(a));
         }
+        case IfThenElse ite -> {
+          if (isInTailPos) {
+            markAsTail(ite);
+            // Note [Analysing Branches in Case Expressions]
```

Comment:
Leftover comment.

Reply:
The comment isn't that "off". Updated in f2d5574ca2